.gitignore

```diff
@@ -0,0 +1,114 @@
+# container ids
+*.running_container_id*
+
+# Created by https://www.gitignore.io/api/python,pycharm
```

Comment:
well, then please add the license information to the .gitignore and include a copy of https://github.com/github/gitignore/blob/master/LICENSE somewhere (maybe renamed to LICENSE.gitignore or so).